### PR TITLE
add dependabot.yml

### PR DESCRIPTION
**Pull request by wmmc88**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    pull-request-branch-name:
+      separator: "/"
+    assignees:
+      - "wmmc88"
+    reviewers:
+      - "wmmc88"
```
Comment (wmmc88) [inline .github/dependabot.yml:25]: @azumnanji  @hamzaMahdi do you want to be added here? its just the pr assignee and reviewers the automatic pr will set.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "daily"
+    pull-request-branch-name:
+      separator: "/"
+    assignees:
+      - "wmmc88"
+    reviewers:
+      - "wmmc88"
```
Comment (wmmc88) [inline .github/dependabot.yml:14]: @azumnanji  @hamzaMahdi do you want to be added here? its just the pr assignee and reviewers the automatic pr will set.

Comment (azumnanji) [reply .github/dependabot.yml:14]: yes add me as a reviewer

**Review by azumnanji — APPROVED** [review]
just add me as reviewer

**Pull request merged**